admin/afs-client-accessd/afs-client-accessd

```diff
@@ -601,6 +611,11 @@ read_config()
 		$newcfg{$_} = $override{$_};
 	}
 
+	if ($newcfg{EXPORT_TIME} < 0 || $newcfg{EXPORT_TIME} > 24) {
+		$newcfg{EXPORT_TIME} = 0;
+	}
+	$newcfg{EXPORT_TIME} = $newcfg{EXPORT_TIME} * 60 * 60;
```

Comment:
I would not convert the new config value here. If you want to do the conversion once, maybe create a new variable?  If feels very side-effectly.